### ISSUE TITLE
White labeling and/or stealth

### ISSUE THREAD

**Issue by Caffeinedesign** [soflyy/oxygen-bugs-and-features#148]
**Describe the feature you'd like to see included in Oxygen.**
I think, being a site builder, so usually used by agencies to make website for customers, would be nice to have a white label system, wich changes plugin name, image, url etc with agency one, i know many agencies i worked with, wich prefer to have white label on a tool they use. (as i read on an old suggestion would be nice to white label "standard dynamic" IDs and classes too

A nice feature would be also a "stealth mode" to hide anything about oxygen, wich can be toggled by a specific url for example

**What are the use cases for this feature?**
Would be good for many agencies wich prefer to have their tools white labeled for both support and less messing by customers.
But i worked with some of them wich think their customers would devaluate their work discovering they used a third party "visual" tool, for example years ago when visual composer was just back-end and shortcode based, i had a collaboration with an agency wich used visual composer, generated tons of shortcodes and made "visual" not visible, so if customers saw pages they just saw thousands of almost unreadable codes

**Examples of this feature or functionality.**
Beaver builder and its addons (for example powerpack) have a white labeling system, you can change plugin and theme names, urls, images etc.
For divi there's divi ghoster wich can both white label and completely hide divi

**Comment by goldpromotion** [comment]
Definitely a must-have if you want to win business from Beaver Builder

**Comment by hawkmusician** [comment]
+1 from me, would really like to see this before using Oxygen in my agency

**Comment by amnither** [comment]
Yes, I believe white label solution is a must thing for agencies.

**Comment by seothemes** [comment]
+1

**Comment by Noz5** [comment]
+1

**Comment by joomapp** [comment]
+2 (just an extra upvote :-p)

**Comment by Alechilles** [comment]
+1

**Comment by M-Marbouh** [comment]
+1

**Comment by fenwikk** [comment]
+1

**Comment by jancikviktor** [comment]
Something new about that? It's a long time from the last reply :)

**Comment by YammYcoding** [comment]
+1

**Comment by cjperes** [comment]
+1

**Comment by abdulfx87** [comment]
+2

**Comment by sadrian80** [comment]
+1 on white labelling.

We had a lot of negative feedback over the years for using various Wordpress builders. Many customers seem to be under the impression, that using a builder tool is "cheap" and "bad practice"; they seem to expect us to deliver custom coded sites, even though we clearly differentiate theme customising, builders and custom coding in our communications. On the other side, we can sell better when our agency branding is in the backend.

**Comment by martinlrg** [comment]
+1

**Comment by uxtherene** [comment]
+1

**Comment by reyadalfaituri** [comment]
I think it is good to have this feature as an option to be enabled or disabled by the site admin. However, If it will cause problems and conflicts with other plugins or performance issues, I think it would be better to have it in a separate extension, not in the core plugin code. 

Recently, this feature become very popular, I know at least 4 theme/page builders that offer white-label features; Beaver Builder, Brizy, Zion Builder, and through third-party plugins such as Divi (Divi Ghost), Elementor (PowerPackelEments), and I am sure there is more.

**Comment by farhanmunim** [comment]
+1

**Comment by thepofo** [comment]
+1

**Comment by powern3rd** [comment]
+1

**Comment by LauraHamburg** [comment]
**+100000000000** 
GIVE US THIS IN 4.0 RELEASE!!!

**Comment by Toastbrot2210** [comment]
+1 on white labelling.
A feature that is absolutely necessary. I would like to use oxygen for my agency.

**Comment by why13ee** [comment]
+1
